pytket/extensions/qujax/qujax_convert.py

```diff
@@ -28,104 +28,58 @@ def _tk_qubits_to_inds(tk_qubits: Sequence[Qubit]) -> Tuple[int, ...]:
     Convert Sequence of pytket qubits objects to tuple of integers qubit indices.
 
     :param tk_qubits: Sequence of pytket qubit object
-        (as stored in pytket.Circuit.qubits).
+        (as stored in ``pytket.Circuit.qubits``).
     :type tk_qubits: Sequence[Qubit]
     :return: Tuple of qubit indices.
-    :rtype: tuple
+    :rtype: Tuple[int]
     """
     return tuple(q.index[0] for q in tk_qubits)
 
 
 def tk_to_qujax_args(
-    circuit: Circuit,
+    circuit: Circuit, symbol_map: Union[bool, dict] = False
 ) -> Tuple[Sequence[str], Sequence[Sequence[int]], Sequence[Sequence[int]], int]:
     """
     Converts a pytket circuit into a tuple of arguments representing
     a qujax quantum circuit.
-    Assumes all circuit gates can be found in qujax.gates.
-    Parameter argument of created function will be ordered as in circuit.get_commands()
-    - pytket automatically reorders some gates, consider using Barriers.
+    Assumes all circuit gates can be found in ``qujax.gates``
+    The ``symbol_map`` argument controls the interpretation of any symbolic parameters
+    found in ``circuit.free_symbols()``.
+
+    - If ``symbol_map`` is ``False``, circuit.free_symbols() will be ignored.
```

Comment:
What happens if there are free symbols in the case symbol_map=False ? It is not clear to me what the "will be ignored" means. If there are symbolic parameters, what happens with them?